client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -40,18 +43,22 @@ class PluginAutomatedTransfer extends Component {
 	};
 
 	componentWillMount() {
-		const { COMPLETE, CONFLICTS } = transferStates;
-		const { isTransferring, transferState } = this.props;
+		const { COMPLETE } = transferStates;
+		const {
+			isTransferring,
+			isTransferFailed,
+			transferState,
+		} = this.props;
 
 		if ( COMPLETE === transferState ) {
 			this.setState( { transferComplete: true } );
-		} else if ( isTransferring || CONFLICTS === transferState ) {
+		} else if ( isTransferring || isTransferFailed ) {
```

Comment:
Ugh!
I've actually spent a fair amount of time researching if "is failed" could be somehow acceptable  (so to keep it near `isAutomatedTransferActive.js` in the selector list), and it is quite incorrect but there are some other software cases that use it.

Gonna change it everywhere from `is` to `has`.